vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/vaadin-grid-flow-selection-column.js

```diff
@@ -80,6 +80,10 @@ import { GridColumnElement } from '@vaadin/vaadin-grid/src/vaadin-grid-column.js
       if (this._grid) {
         this._grid.addEventListener('select', this._boundOnSelectEvent);
         this._grid.addEventListener('deselect', this._boundOnDeselectEvent);
+        // Prevent flickering when using the Lumo compact theme
+        if (this._grid.theme && this._grid.theme.split(' ').indexOf('compact') > -1) {
+            this.width = '40px';
```

Comment:
It would be nice to add a comment explaining how we got this magic number.

Something like this:
```js
/* 
This value of 40 px includes:
calc(var(--lumo-space-s) * 2) - cell padding = 16px
calc(1em + 2px) - checkbox width = 18px
calc(0.1875em * 2) - checkbox margin = 6px
*/

Reply:
Done